### ISSUE TITLE
Tailscale 1.34.1. Login doesn't work

### ISSUE THREAD

**Issue by tchaton** [tailscale/tailscale#6758]
### What is the issue?

Hey there,

I have just installed Tailscale 1.34.1 on MacOs Montery and it seems the login isn't working properly. It gets blocked in loading... phase and sign-in isn't available. 

### Steps to reproduce

Not sure.

### OS

macOS

### OS version

12.5.1

### Tailscale version

1.34.1

**Comment by bradfitz** [comment]
Did Tailscale ever previously work for you?

From App Store or the standalone download?

Any other antivirus/firewall/Screen Time content filtering enabled on that machine?

/cc @mihaip @nickoneill

**Comment by tc60045** [comment]
I'm on Ventura, have used TailScale for a year on multiple Macs.  After updating to OS 13.1, Tailscale will not log in -- that is to say, the option to Log In is available, but clicking on it does nothing.  <dead in the water now...>

<img width="207" alt="image" src="https://user-images.githubusercontent.com/2856318/207948609-71f4d0b4-78f9-4ea0-94b6-2e75842bd48d.png">

**Comment by tchaton** [comment]
Here is a loom recording of the issue I am facing: https://www.loom.com/share/5f59342fdafe4c12bec44f686fdf45db

**Comment by tchaton** [comment]
@bradfitz  Yes, I worked on my previous computer. This is a fresh installation.

I used brew install -cask tailscale to install the App.

**Comment by nickoneill** [comment]
Hi @tchaton, can you restart tailscale and see if it shows you a "allow this system extension" system dialog? If so, click it to reach the Privacy and Security pane of the preferences app, and look for an "Allow" button indicating that you want the network extension for Tailscale to run. The onboarding screen should indicate that the VPN has started after this.

**Comment by tchaton** [comment]
Hey @nickoneill I installed TailScale from the App Store and it worked just fine. It seems the installation through brew was the problem.

**Comment by vanayr** [comment]
Just a heads up, I had the same issue.  I had to update, quit the app, re-launch, then log in.  Seemed to fix the issue for me running 13.1 (22C65)

**Comment by yostyle** [comment]
same on 1.36.2

**Comment by DentonGentry** [comment]
> I had to update from the app store, quit the app, re-launch, then log in. Seemed to fix the issue for me running 13.1 (22C65).

These are almost always due to the Network Extension (App Store) or System Extension (downloadable) getting into a state that it won't unload to allow the new version to take over.

**Comment by Empty2k12** [comment]
I ran into this issue when setting up a new Mac. Installed from brew directly after first boot, never opened App Store on this machine.
